presto-main/src/main/java/io/prestosql/metadata/FunctionRegistry.java

```diff
@@ -706,6 +718,16 @@ public final synchronized void addFunctions(List<? extends SqlFunction> function
         this.functions = new FunctionMap(this.functions, functions);
     }
 
+    private static boolean isNotEqualsOrHashCodeOperator(SqlFunction function)
```

Comment:
This method name is confusing. I can't tell (without reading the code) whether it's "is NOT_EQUALS or HASH_CODE operator" or "is not EQUALS or HASH_CODE operator". Maybe reverse the order to clarify: `isNotHashCodeOrEqualsOperator`